pkg/build/api/types.go

```diff
@@ -14,6 +14,8 @@ const (
 	DeprecatedBuildLabel = "build"
 	// BuildLabel is the key of a Pod label whose value is the Name of a Build which is run.
 	BuildLabel = "openshift.io/build.name"
+	// DefaultNamespaceLabel is the key of a Build label, whose values are SourceInfo of the build.
+	DefaultNamespaceLabel = "io.openshift."
```

Comment:
at a minimum this needs to be clearly documented (and perhaps even named) to indicate it is a docker LABEL and not a k8s label.